docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -25,19 +25,19 @@ Download the *dirac-install* script from::
 
 Choose the directory where you want to install the DIRAC software and run the dirac-install script from
 this directory giving the appropriate version of the DIRAC release,
-and, the version of the "lcgBundle" (with "-g" option) that you want to use::
+and the version of the "lcgBundle" (with "-g" option, not required for MacOS 10.14 users) that you want to use::
```

Comment:
It is not that lcg bundles are not required, there are no lcg bundles for MacOS (any version of that OS, and also for other operating systems). It can also be that for certain communities the lcg-bundles are pre-defined, in which case the lcg-bundles need to be disabled via a different flag.

Reply:
I have a doubt regarding this...According to what I have read and understood, lcgbundles are mainly statically-compiled libraries used for interacting with grid environments, which is the base for distributed computing, then how is it possible that there are no lcgbundles for other operating systems as well or is it that they are pre-defined by most of the communities??

Reply:
Simply put, nobody has bothered (or managed) to compile these libraries for MacOS, which is why they are unavailable. This means those users cannot directly interact with the grid storage elements.

I changed the phrase about the pre-defined lcg-bundles to clarify that this refers to the lcg-bundles version, which would be attempted to be downloaded, and give error messages.

Reply:
We don't (we can't, in fact) support native clients on MacOS.

Reply:
Can you please update according to the comments -> not available for Mac users ?

Reply:
@atsareg  I'll be updating it today...